app/assets/stylesheets/_post.scss

```diff
@@ -22,76 +27,219 @@
+      width: 100%;
+      max-width: $content-max-width;
+    }
 
-  &__subtitle {
-    font-size: 1.25rem;
-    padding-bottom: 20px;
-  }
+    &__taglist {
+      list-style-type: none;
+      display: flex;
+      width: 100%;
+      flex-flow: row-reverse;
+      margin: 1rem auto;
 
-  &__tags {
-    list-style-type: none;
-    display: flex;
-  }
+      @include media("small") {
+        width: 50%;
+        max-width: 450px;
+        flex-flow: row;
+        height: 60px;
+        margin: auto 0 2rem auto;
+      }
+    }
+
+    &__tag {
+      margin: auto .5rem;
+      font-size: .8rem;
+      padding: .3rem .8rem;
+      background: $color-black;
+      color: $color-white;
+      border-radius: 1.2rem;
+      height: 1.6rem;
+
+      &:first-of-type {
+        margin-left: 0;
+      }
+    }
+
+    &__info {
+      margin: 0 auto auto auto;
+      width: 100%;
+      display: flex;
+      flex-flow: row wrap;
+      height: 60px;
+
+      @include media("small") {
+        width: 50%;
+        max-width: 450px;
+        margin: auto auto 2rem 0;
+      }
+
+      &__text {
+        width: calc(100% - 80px);
+        text-align: right;
+        margin: auto;
+      }
+
+      &__author {
+        font-size: 1rem;
+        font-weight: 300;
+      }
+
+      &__author-picture {
+        width: 60px;
+        height: 60px;
+        object-fit: cover;
+        margin-left: 20px;
+        border-radius: 50%;
+      }
+
+      &__date {
+        font-size: .7rem;
+        font-weight: 300;
+      }
+    }
 
-  &__tag {
-    margin: 0 .5rem;
-    font-size: .8rem;
-    padding: .3rem .8rem;
-    background: #555;
-    color: #111;
-    border-radius: 1.2rem;
   }
 
   &__body {
-    padding-left: 15px;
-    padding-right: 15px;
+    margin: auto;
+    width: 90%;
+    max-width: $content-max-width;
+
+    img {
+      max-width: 100%;
+    }
 
-    &__subtitle {
-      margin-bottom: 10px;
-      margin-top: 40px;
+    h1 {
+      display: none;
     }
 
-    &__text {
+    ul,
+    ol {
+      list-style-position: inside;
+    }
+
+    li {
+      font-weight: 300;
+      color: $color-black;
       font-size: 1.15rem;
-      line-height: 1.7;
-      margin-bottom: 20px;
-      margin-top: 20px;
+
+      &:not(:last-of-type) {
+        margin-bottom: .8rem;
+      }
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+      font-weight: bold;
+      color: $color-black;
+      margin-top: 3rem;
+      margin-bottom: 1.5rem;
+    }
+
+    h2 {
+      font-size: 1.3rem;
+    }
+
+    h3,
+    h4 {
+      font-size: 1.1rem;
+    }
+
+    p {
+      font-size: 1rem;
+      line-height: 1.7rem;
+      margin-bottom: 1.5rem;
+      margin-top: 1.5rem;
+      color: $color-black;
+      font-weight: 300;
+    }
+
+    code {
+      position: relative;
+      color: $color-ice;
+      display: block;
+      background: #333;
+      padding: 1rem;
+      border-left: 6px solid #f73d9d;
+      box-shadow: 1px 1px 10px rgba(0,0,0,.3);
```

Comment:
Poderiamos continuar usando classes e o que for genérico passar para um outro arquivo e deve ser feita na tag, passar para outro arquivo

Reply:
Passar as cores usadas aqui para váriaveis tbm 💅

Reply:
- nao entendi isso ```Poderiamos continuar usando classes e o que for genérico passar para um outro arquivo e deve ser feita na tag, passar para outro arquivo``` - a que você se referiu exatamente?
- deixei passar o #333 nao usar variavel, vou colocar!
- vou adicionar o border-left e a cor da sombra pra variaveis

Reply:
- deixei de colocar jogar a cor do border-left em variavel por enquanto pois vou criar uma issue específica pra estilizar code blocks
- deixei de colocar as shadows em variaveis por enquanto pra criar uma issue especifica pra elas